### ISSUE TITLE
extend to other datasets

### ISSUE THREAD

**Issue by Richardg706** [hnuzhy/BPJDet#22]
Thank you very much for your excellent work. I came across the paper PBADET related to your team, which mentioned that the detection could be extended to animals. Could you please confirm if this project can achieve similar results?
![CZC `NM6B%OB64M9 0_)@WK](https://github.com/hnuzhy/BPJDet/assets/104958583/abc1c517-fec0-4652-9a12-6f4501005d41)

**Comment by hnuzhy** [comment]
Of course, both BPJDet and PBADet can achieve the joint detection and association of body-parts, but with different designs. The main object can be human, animal and any other things (e.g., Car-Wheel, Table-Foot and Clothing-Button), as long as you have the corresponding datasets. For more details of building the training and testing datasets for animals, you can refer the journal version of BPJDet in https://arxiv.org/pdf/2304.10765v2.pdf. Please see Sections `5.1.1` and `5.3.5`
